src/gobeventproducer/producer.py

```diff
@@ -112,16 +165,15 @@ def _get_tables_to_reflect(self):
         relation_tables = [gob_model.get_table_name('rel', rel_table) for rel_table in relations.values()]
 
         return [
-                   'events',
-                   gob_model.get_table_name(self.catalogue, self.collection)
-               ] + relation_tables
+            'events',
+            gob_model.get_table_name(self.catalogue, self.collection)
+        ] + relation_tables
 
     def _init_gob_db_session(self):
-        """Inits db session for gob db (to access events)
+        """Init db session for gob db (to access events).
 
         :return:
         """
-
         engine = create_engine(URL(**GOB_DATABASE_CONFIG), connect_args={'sslmode': 'require'})
         self.gob_db_session = Session(engine)
         meta = MetaData()
```

Comment:
Seeing some (unrelated) warnings during reflection here:
SAWarning: Skipped unsupported reflection of expression-based index rel_bag_wps_bag_dsr_dec11bbe_ed3f22b3eec2fb035647f924a5b2136e
2023-02-27T11:00:50.220595256Z   meta.reflect(engine, only=self._get_tables_to_reflect())